### PR TITLE
add SubmitLog and CreateLog overloads using the LogLevel enum

### PR DESCRIPTION
**Pull request by xbelt**
I find this feature very useful. Choosing an option in an enum is easier than remembering the correct strings

### CODE REVIEW

File: Source/Shared/Extensions/ClientExtensions.cs [hunk 1]
```diff
@@ -63,6 +64,18 @@ public static class ClientExtensions {
...
 
         /// <summary>
+        /// Submits a log message event.
+        /// </summary>
+        /// <param name="client">The client instance.</param>
+        /// <param name="source">The log source.</param>
+        /// <param name="level">The log level.</param>
+        /// <param name="message">The log message.</param>
+        public static void SubmitLog(this ExceptionlessClient client, string source, string message, LogLevel level)
+        {
```
Comment (niemyjski) [inline Source/Shared/Extensions/ClientExtensions.cs]: Can you please put the brackets on the same line (consistency) :).

File: Source/Shared/Extensions/ClientExtensions.cs [hunk 2]
```diff
@@ -98,6 +111,24 @@ public static class ClientExtensions {
...
+        public static EventBuilder CreateLog(this ExceptionlessClient client, string source, string message, LogLevel level)
+        {
+            var builder = client.CreateLog(source, message);
+
+            var stringLogLevel = level.ToString();
+            if (!String.IsNullOrWhiteSpace(stringLogLevel))
+                builder.AddObject(stringLogLevel.Trim(), Event.KnownDataKeys.Level);
+
+            return builder;
+        }
```
Comment (niemyjski) [inline Source/Shared/Extensions/ClientExtensions.cs]: I think this method should just return CreateLog(client, string, string, string) and call level.ToString() as the last argument. This way we don't have duplicated code that does the same thing :)

**Comment by niemyjski** [comment]
Thanks for your pull request, I've made some comments :). Can you also please make sure you sign our CLA: http://www.clahub.com/agreements/exceptionless/Exceptionless

**Comment by xbelt** [comment]
Thank you for your comments!
I needed to recreate the pull request because I was working on the master branch. I already signed the CLA.
So I'll close this pull request now

**Pull request closed without merging**